other.lua

```diff
@@ -85,60 +85,59 @@ function boat:on_step(dtime)
 			self.v = self.v-0.03
 		end
 		if ctrl.left then
-			self.object:setyaw(self.object:getyaw()+math.pi/120+dtime*math.pi/120)
+			self.object:set_yaw(self.object:get_yaw()+math.pi/120+dtime*math.pi/120)
 		end
 		if ctrl.right then
-			self.object:setyaw(self.object:getyaw()-math.pi/120-dtime*math.pi/120)
+			self.object:set_yaw(self.object:get_yaw()-math.pi/120-dtime*math.pi/120)
 		end
 	end
 	local s = get_sign(self.v)
 	self.v = self.v - 0.02*s
 	if s ~= get_sign(self.v) then
-		self.object:setvelocity({x=0, y=0, z=0})
+		self.object:set_velocity({x=0, y=0, z=0})
 		self.v = 0
 		return
 	end
 	if math.abs(self.v) > 4.5 then
 		self.v = 4.5*get_sign(self.v)
 	end
 	
-	local p = self.object:getpos()
+	local p = self.object:get_pos()
 	p.y = p.y-0.5
 	if not is_water(p) then
 		if minetest.registered_nodes[minetest.get_node(p).name].walkable then
```

Comment:
Should be fixed somewhen to not crash when driving over unknown nodes.

Reply:
How about an issue then... :smile: